whitepaper/whitepaper.md

```diff
@@ -124,13 +134,235 @@ Blockchain technologies are growing in usage, but fragmentation is a big problem
 
 We propose a protocol and it's implementation to connect as many of them as possible in an attempt to solve the fragmentation problem by creating a heterogeneous system architecture <sup>[1](#7-references)</sup>.
 
-# 2. Example Use Cases
+# 2. Introduction to Blockchain Interoperability
+
+There are two inherent problems<sup>[a](#22-footnotes-introduction)</sup> that have to be solved when connecting different blockchains: 
+* How to provide a proof of the networkwide<sup>[b](#22-footnotes-introduction)</sup> ledger state of a connected blockchain from the outside?
+* How can other entities verify a given proof of the state of a connected blockchain
+from the outside? 
```

Comment:
If a blockchain uses Merkle trees to store its state than anyone (even un-trusted parties) can provide a proof which can be verified through the Merkle tree root and techniques like simple payment verification without trusted third parties. But if the blockchain does not have an support to verify its state from the outside than trusted third parties are often needed. 

Yes, Cactus is mainly about technical interoperability. There was once the discussion about using the Read_the_docs framework for documenting Cactus (https://readthedocs.org/). It might give us the opportunity to include chapters/paragraphs focusing on areas and use cases beyond technical blockchain interoperability. The white paper is probably focusing mainly on technical blockchain interoperability since this is the main deliverable of Cactus.

Reply:
Yes!